osu.Game.Tests.Android/osu.Game.Tests.Android.csproj

```diff
@@ -28,6 +28,7 @@
     <ProjectReference Include="..\osu.Game\osu.Game.csproj" />
   </ItemGroup>
   <ItemGroup Label="Package References">
+    <PackageReference Include="Bogus" Version="35.6.2" />
```

Comment:
If this lives, not sure why it's only added to the android tests project, iOS test project will need it too presumably

Reply:
Hmmm. We haven't been building iOS tests until now... Will circle back around to this after https://github.com/ppy/osu/pull/32437 is merged.